new_listings_scraper.py

```diff
@@ -14,15 +15,25 @@ def get_last_coin():
     latest_announcement = latest_announcement.json()
     latest_announcement = latest_announcement['data']['articles'][0]['title']
 
-    # Binance makes several annoucements, irrevelant ones will be ignored
+    found_coin = re.findall('\(([^)]+)', latest_announcement)
```

Comment:
If an announcement will ever contain any text in parenthesis (because why not?) it would be matched by this regex. Perhaps consider replacing it with: `\(([^a-z\s)]+)\)` to disallow whitespaces and lowercase letters? Just an idea.